### PR TITLE
US120977 - Add Notification Email field to Quiz

### PR DESCRIPTION
**Pull request by wongvincent**
https://rally1.rallydev.com/#/detail/userstory/438838251528?fdp=true

### CODE REVIEW

File: test/d2l-activity-editor/d2l-activity-quiz-editor/state/quiz.spec.js [hunk 1]
```diff
@@ -93,10 +97,19 @@ describe('Quiz', function() {
 		await quiz.fetch();
 
 		const password = 'super-secret-password';
-
 		quiz.setPassword(password);
 
 		expect(quiz.password).to.equal(password);
 	});
 
+	it('setPassword', async() => {
```
Comment (mdulat) [inline test/d2l-activity-editor/d2l-activity-quiz-editor/state/quiz.spec.js:105]: Should be called `setNotificationEmail` (not password)

**Review by mdulat — APPROVED** [review]

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-notification-email-editor.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+	render() {
+
+		const entity = store.get(this.href);
+		if (!entity) {
+			return html``;
+		}
+
+		return html`
+			<div id="assignment-notification-email-container">
```
Comment (mostlyfabulous) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-notification-email-editor.js:38]: Should `id` be prefixed with `quiz` vs `assignment`?

**Review by mostlyfabulous — DISMISSED** [review]
LGTM

**Review by mostlyfabulous — APPROVED** [review]
👍

**Review by mdulat — APPROVED** [review]

**Pull request merged**